Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -481,7 +481,22 @@ private void MeleeDamage(FPSWeapon weapon)
 
                 // TODO: Use correct damage based on weapon and swing type
                 // Just using fudge values during development
```

Comment:
Are you planning on addressing the rest of the TODO after this, or is that something I could pick up?